src/w3d/math/quaternion.cpp

```diff
@@ -54,17 +53,17 @@ Quaternion::Quaternion(const Vector3 &axis, float angle)
 
 void Quaternion::Rotate_X(float theta)
 {
-    *this = (*this) * Quaternion(Vector3(1, 0, 0), theta);
+    *this = (*this) * Quaternion(Vector3(1.f, 0.f, 0.f), theta);
 }
 
 void Quaternion::Rotate_Y(float theta)
 {
-    *this = (*this) * Quaternion(Vector3(0, 1, 0), theta);
+    *this = (*this) * Quaternion(Vector3(0.f, 1.f, 0.f), theta);
 }
 
 void Quaternion::Rotate_Z(float theta)
 {
-    *this = (*this) * Quaternion(Vector3(0, 0, 1), theta);
+    *this = (*this) * Quaternion(Vector3(0.f, 0.f, 1.f), theta);
```

Comment:
it looks weird not using one unit after the decimal point for a floating point

Reply:
It is valid code though. Can append a 0 if so desired for style.

Reply:
I think that would look better

Reply:
Added the zeros.